docs/data-analysis.md

```diff
@@ -1,164 +1,108 @@
 # Analyzing bot data
 
-After performing backtests, or after running the bot for some time, it will be interesting to analyze the results your bot generated.
+You can analyze the results of backtests and trading history easily using Jupyter notebooks. A sample notebook is located at `user_data/notebooks/analysis_example.ipynb`. For usage instructions, see [jupyter.org](https://jupyter.org/documentation).
 
-A good way for this is using Jupyter (notebook or lab) - which provides an interactive environment to analyze the data.
+*Pro tip - Don't forget to start a jupyter notbook server from within your conda or venv environment or use [nb_conda_kernels](https://github.com/Anaconda-Platform/nb_conda_kernels)*
 
-The following helpers will help you loading the data into Pandas DataFrames, and may also give you some starting points in analyzing the results.
+## Example snippets
 
-## Strategy development problem analysis
-
-Debugging a strategy (are there no buy signals, ...) can be very time-consuming.
-FreqTrade tries to help you by exposing a few helper-functions, which can be very handy.
-
-It's recommended using Juptyer Notebooks for analysis, since it offers a dynamic way to rerun certain parts of the code.
-
-The following is a full code-snippet, which will be explained by both comments, and step by step below.
+### Load backtest results into a pandas dataframe
 
 ```python
-# Some necessary imports
-from pathlib import Path
-
-from freqtrade.data.history import load_pair_history
-from freqtrade.resolvers import StrategyResolver
-# Define some constants
-ticker_interval = "5m"
-
-# Name of the strategy class
-strategyname = 'Awesomestrategy'
-# Location of the strategy
-strategy_location = '../xmatt/strategies'
-# Location of the data
-data_location = '../freqtrade/user_data/data/binance/'
-# Only use one pair here
-pair = "XRP_ETH"
-
-### End constants
+# Load backtest results
+df = load_backtest_data("user_data/backtest_data/backtest-result.json")
```

Comment:
this section (and the following one about live-data) looks odd when looked at in the documentation (or within the file - it's not obvious in the github diff). 

it uses `load_backtest_data` without ever importing it. 
The imports are a few code-blocks below - but that's not obvious and forces users to jump up/down in this document.
i would either move this section below the imports, or keep the import in this code-block (it won't hurt and provides a simple copy/pasteable method if someone doesn't want to use the sample notebook).
My personal preference would be the latter ... so keeping the related import in this cell / code block.

Reply:
I see what you mean. I was trying to reflect the notebook cells in the documentation, but that doesn't exactly work, because the documentation page is not considered as a whole script the way a notebook page is.

I added the imports to the code cells in the documentation, for the first 2 snippets. But left the constants and imports together for the strategy analysis section. Also fixed a few other inconsistencies.